### ISSUE TITLE
Man, What can I say!

### ISSUE THREAD

**Issue by Captain-Rhino** [yuweihao/MambaOut#93]
牢大！

**Comment by wang-hanmo** [comment]
确实。另外科比·布莱恩特，出生于美国宾夕法尼亚州费城，美国已故篮球运动员，司职得分后卫/小前锋1996年NBA选秀，科比于第1轮第13顺位被夏洛特黄蜂队选中并被交易至洛杉矶湖人队，整个NBA生涯都效力于洛杉矶湖人队；共获得5次NBA总冠军、1次NBA常规赛MVP、2次NBA总决赛MVP、4次NBA全明星赛MVP、2次NBA赛季得分王；共入选NBA全明星首发阵容18次、NBA最佳阵容15次（其中一阵11次、二阵2次、三阵2次）、NBA最佳防守阵容12次（其中一阵9次、二阵3次，阵亡一次
